### PR TITLE
feature: add `install_repo` property to `draw-zmk.yml`

### PR DESCRIPTION
**Pull request by SethMilliken**
Facilitates testing `keymap-drawer` PRs from a fork.

### CODE REVIEW

**Comment by SethMilliken** [comment]
I was not able to see a way to automatically determine the host repo of the `draw-zmk.yml` workflow. At runtime, the `github` context is for e.g. the `zmk-config` repo it is running against. So you could do something hacky like use `${{ github.repository_owner }}`, but that would not be as flexible as the `install_repo` property solution here (and is aesthetically ugly IMO).

File: .github/workflows/draw-zmk.yml [hunk 1]
```diff
@@ -44,6 +44,11 @@ on:
         default: false
         required: false
         type: boolean
+      install_repo:
+        description: 'Install keymap-drawer from a git repo. Primarily for testing changes using a keymap-drawer fork.'
+        default: 'caksoylar/keymap-drawer'
+        required: false
+        type: string
```
Comment (caksoylar) [inline .github/workflows/draw-zmk.yml:51]:
```suggestion
      install_repo:
        description: 'Install keymap-drawer from a different git remote, primarily for testing changes using a keymap-drawer fork. Ignored if `install_branch' is unset/empty.
        default: 'https://github.com/caksoylar/keymap-drawer.git'
        required: false
        type: string
```

Tweaked the description a bit and made it the full remote URI, so you can use other git forges.

File: .github/workflows/draw-zmk.yml [hunk 2]
```diff
@@ -92,7 +97,7 @@ jobs:
 
       - name: Install keymap-drawer (git)
         if: inputs.install_branch != ''
-        run: python3 -m pip install "git+https://github.com/caksoylar/keymap-drawer.git@${{ inputs.install_branch }}"
+        run: python3 -m pip install "git+https://github.com/${{ inputs.install_repo }}@${{ inputs.install_branch }}"
```
Comment (caksoylar) [inline .github/workflows/draw-zmk.yml:100]:
```suggestion
        run: python3 -m pip install "git+${{ inputs.install_repo }}@${{ inputs.install_branch }}"
```

File: .github/workflows/draw-zmk.yml (hunk 1 above)
Comment (caksoylar) [reply .github/workflows/draw-zmk.yml:51]: Can you also move this to below `install_branch`, since this depends on the former?

**Review by caksoylar — COMMENTED** [review]
Thanks, I agree with your solution approach, noted a few tweaks.

**Review by caksoylar — APPROVED** [review]

**Comment by SethMilliken** [comment]
Updated and successfully tested from my fork.

**Pull request merged**